plugins/ci/skills/payload-snapshot/scripts/payload_snapshot.py

```
@@ -1952,6 +2200,37 @@ def _check_gcloud() -> bool:
 
+def _check_podman() -> bool:
+    """Verify that podman is available and functional.
+
+    Goes beyond --version: runs 'podman info' which exercises the runtime,
+    storage, and namespace setup.  This catches cases where podman is
+    installed but cannot operate (e.g. nested containers without
+    appropriate privileges).
+    """
+    try:
+        result = subprocess.run(
+            ["podman", "info"],
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)